x-pack/solutions/security/plugins/security_solution/public/detections/configurations/security_solution_detections/render_cell_value.tsx

```diff
@@ -84,22 +80,21 @@ export const CellValue = memo(function RenderCellValue({
       !isDetails,
     [columnId, isDetails, rowIndex, tableType]
   );
-  const { browserFields } = useSourcererDataView(sourcererScope);
-  const browserFieldsByName = useMemo(() => getAllFieldsByName(browserFields), [browserFields]);
-  const getTable = useMemo(() => dataTableSelectors.getTableByIdSelector(), []);
-  const license = useLicense();
-  const viewMode =
-    useDeepEqualSelector((state) => (getTable(state, tableId ?? '') ?? tableDefaults).viewMode) ??
-    tableDefaults.viewMode;
-
-  const gridColumns = useMemo(() => {
-    return getColumns(license);
-  }, [license]);
-
-  const columnHeaders = useMemo(() => {
-    return viewMode === VIEW_SELECTION.gridView ? gridColumns : eventRenderedViewColumns;
-  }, [gridColumns, viewMode]);
+  const cellValueContext = useContext(AlertTableCellContext);
 
+  if (!cellValueContext) {
```

Comment:
Open to another route for notifying devs that the wrapper is needed. Ideally we don't need a context wrapper and can do this via redux, or some other higher state, but this a simple perf improvement for the time being.